build.gradle

```diff
@@ -138,6 +138,14 @@ publishing {
                 }
 
                 developers {
+                    developer {
+                        name = "Vinh Vo"
+                        email = "vinhvd@gmail.com"
+                    }
```

Comment:
@vinhvd Hi!

As written in the pull-request description, we're trying to nominate developers who have contributed a lot in this Embulk plugin.

Because you're the main developer of this Embulk plugin (in 2018), we wanted to have you in the list.

I picked up your name and email from the Git commit log and/or your public GitHub user page. Please let me know if you don't want, or if you want to be listed in another name/email.